### ISSUE TITLE
Mac OS: `script: illegal option -- e`

### ISSUE THREAD

**Issue by taminomara** [esdmr/fish-completion#201]
Hi! I'm trying to use your extension with VSCode 1.81.1 running on mac os 13.4.1. No completions show up, and I'm getting the following errors in the output window:

```
Failure: Error: Command failed with exit code 1: script -e -q -c fish -iPC 'source $_FISH_COMPLETION_WORKER' /dev/null
script: illegal option -- e
usage: script [-adkpqr] [-t time] [file [command ...]]
```

With Mac OS, `-e` and `-c` are always implied, and the order of arguments is `script <file> <command>...`, so you need to do the following:

```bash
script -q /dev/null fish ...
```

For now, I've been able to make an ad-hoc fix by placing the following `script` python file into my path:

```python
#! /usr/bin/env python

import os
import subprocess
import sys

if " ".join(sys.argv[1:]) == "-e -q -c fish -iPC 'source $_FISH_COMPLETION_WORKER' /dev/null":
    args = (
        "/usr/bin/script -q /dev/null fish -iPC source".split()
        + [os.environ['_FISH_COMPLETION_WORKER']]
    )
    exit(subprocess.run(args).returncode)
else:
    exit(subprocess.run(["/usr/bin/script"] + sys.argv[1:]).returncode)

```

**Comment by esdmr** [comment]
Hello! I’ve tried to fix this in #202, but I can’t test it since I don’t own a macOS.

I’ve attached a VSIX file built from that PR. Can you verify that it’s fixed, please?

[fish-completion-0.2.1-bugfix.201+4c25c29.tar.gz](https://github.com/esdmr/fish-completion/files/12503808/fish-completion-0.2.1-bugfix.201%2B4c25c29.tar.gz)

**Comment by esdmr** [comment]
Hi! I’ve released v0.2.1 with that PR. If it still doesn’t work, please reopen this issue. Thanks!
